tests/testthat/_snaps/LongitudinalSteinFojo.md

```diff
@@ -29,10 +29,10 @@
           lm_sf_mu_bsld ~ normal(mu = 4.09434, sigma = 1)
           lm_sf_mu_ks ~ normal(mu = -0.69315, sigma = 1)
           lm_sf_mu_kg ~ gamma(alpha = 2, beta = 1)
-          lm_sf_omega_bsld ~ lognormal(mu = -1.60944, sigma = 1)
-          lm_sf_omega_ks ~ lognormal(mu = -1.60944, sigma = 1)
-          lm_sf_omega_kg ~ lognormal(mu = -1.60944, sigma = 1)
-          lm_sf_sigma ~ normal(mu = 0, sigma = 1)
+          lm_sf_omega_bsld ~ lognormal(mu = -1.60944, sigma = 1) T[0, ]
```

Comment:
@gowerc 
For lognormal and gamma does the truncation come automatically? It seems to me it shouldn't do anything in this case.